tmpl/package.json

```diff
@@ -15,7 +15,7 @@
       "make_targets": {
         "win32": ["squirrel"],
         "darwin": ["zip"],
-        "linux": ["deb", "rpm"]
+        "linux": ["deb", "rpm", "flatpak"]
```

Comment:
Hmmm. I would say yes, if it wasn't such a pain to install flatpak *and* given how long it takes to build one of those packages.